### PR TITLE
LLVM 7 update

### PR DESCRIPTION
**Pull request by PurplePachyderm**
- In this version, Fortran support has been explicitely disabled (this is a temporary solution before we make sure that flang instead of dragonegg works correctly).
- The current Dockerfile installs LLVM 3.4. Because this PR uses LLVM 7, the Dockerfile has to be modified to get LLVM 7 (notably for the CI to pass). This is probably best to do this before merging, the easiest way would probably be to download the binary releases from the LLVM website, since Ubuntu 14.04 doesn't have LLVM 7 packages.

### CODE REVIEW

**Comment by pablooliveira** [comment]
@PurplePachyderm: this looks great. Before merging could you please fix the conflicts ?

**Comment by PurplePachyderm** [comment]
Done!

File: INSTALL.md [hunk 1]
```diff
@@ -2,11 +2,10 @@
...
 Please ensure that CERE's dependencies are installed on your system:
 
-  * LLVM, clang, dragonegg and opt 3.5, 3.4 or 3.3, http://llvm.org/releases/download.html#3.5.2
+  * LLVM, clang and opt 7, http://llvm.org/releases/download.html#7.0.0
 
-  * gcc, gfortran (for Fortran support)
-    We recommend to use gcc-4.7 we have experienced some problems with dragonegg 4.8
-    version.
+  * gcc
+    We recommend to use gcc-4.7
```
Comment (pablooliveira) [inline INSTALL.md]:
```suggestion
```

Comment (pablooliveira) [reply INSTALL.md]: This is not needed since dragonegg has been removed.

**Review by pablooliveira — APPROVED** [review]
LGTM

**Pull request merged**